core/spec/models/spree/taxon_spec.rb

```diff
@@ -26,6 +26,14 @@
       expect(taxon.permalink).to eql "ruby-on-rails"
     end
 
+    context "updating a taxon permalink" do
+      it 'parameterize permalink correctly' do
+        taxon.save!
+        taxon.update_attributes(permalink: 'spécial&charactèrs')
+        expect(taxon.permalink).to eql "special-characters"
+      end
+    end
```

Comment:
This change also has effects on permalinks of children. Can we test that it works with children as well?

Reply:
ok, I'll add this test.

Reply:
Test added

Reply:
I think the test you added is testing that the permalink is correctly saved when it has a parent taxon. I mean to test when the taxon with special chars has children instead (like `spécial&charactèrs/child` for example) so we can see the parameterization reflected on those taxons as well. Anyway the test you added can stay, better to have that as well. 🙂

Reply:
I added 2 more tests